CONTRIBUTING.rst

```diff
@@ -10,285 +10,14 @@ etiquette guidelines. For more details please see the `CODE_OF_CONDUCT.md file
 <https://github.com/mozilla-services/socorro/blob/main/CODE_OF_CONDUCT.md>`_.
 
 
-Bugs
-====
+Development, conventions, etc
+=============================
```

Comment:
I moved a lot of material from CONTRIBUTING to the regular docs. That way documentation around code conventions, workflow, etc are all in one place rather than in a couple of different places.